lms/views/api/blackboard/files.py

```diff
@@ -18,6 +18,13 @@
     r"blackboard:\/\/content-resource\/(?P<file_id>[^\/]*)\/"
 )
 
+# The maxiumum number of paginated requests we'll make before returning.
+PAGINATION_MAX_REQUESTS = 25
```

Comment:
Holy moley... I think this equates to 5000 files. How long does each round take?

Reply:
Yeah, it's the same as we do for Canvas: 25 * 200 files. I can't remember why we chose that for Canvas. We might want to reduce it to say 10